src/libraries/System.Private.CoreLib/src/System/AppContext.Browser.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static partial class AppContext
+    {
+        private static string GetBaseDirectoryCore()
+        {
+            // GetEntryAssembly().Location returns an empty string for wasm
+            // Until that can be easily changed, work around the problem here.
+            return "/";
```

Comment:
Should we add explicit test for this?

Reply:
Good question.  I think it's acceptable to just rely on the tests that use this property to pass.